models.py

```diff
@@ -189,7 +189,8 @@ def __init__(
 		normalize_features_eps = torch.finfo(torch.float16).tiny,
 		normalize_features_track_running_stats = False,
 		normalize_features_legacy = True,
-		normalize_features_temporal_mask = True
+		normalize_features_temporal_mask = True,
+		check_time_dim_padded = False,
```

Comment:
let's make it true by default? any reasons/cases for not enabling this by default?

Reply:
If we transcribe some audios with transcribe.py then we have no guaranty that the input signal is padded.

Reply:
then let's enable it by default and only pass `check_time_dim_padded = False` in transcribe.py

Reply:
done